Makefile.am

```diff
@@ -338,6 +407,7 @@ janus_pp_rec_CFLAGS = \
 
 janus_pp_rec_LDADD = \
 	$(POST_PROCESSING_LIBS) \
+	libjanus.la \
```

Comment:
Why? The post-processor does not have the same dependencies as the Janus executable, most of the stuff there is not needed.